service/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,14 @@
+landingzone:
```

Comment:
I am a bit confused how we're able to run the LandingZoneApiControllerTest without this db config; regardless this is needed for running integration tests in LZS locally.

Reply:
This wasn't needed after all, it's been removed.